collect_app/src/main/java/org/odk/collect/android/utilities/StringUtils.java

```diff
@@ -158,5 +158,25 @@ public static boolean isBlank(String string) {
 
         return true;
     }
+
+    private static CharSequence trim(CharSequence text) {
```

Comment:
Could we just use `String#trim` for this or is there an extra case we need covered? Looks like the test passes using `toString().trim()` but I might be missing something/test needs to be stricter.

Reply:
We can't  because we need to return `CharSequence` not string (to use it with `TextView.setText()`) otherwise html styling won't work